README.md

```diff
@@ -22,6 +25,21 @@ ILP_SECRET='Z4ZwCWbuo18eMrS1Tgn5L0FPUgK/2pKvgdvZzMogPw4' \
 docker-compose up -d
 ```
 
+> Log in to your ilp-kit
+
+First, wait for about 20 minutes! Until then, there will be nothing to see yet on your ilp-kit URL, or just an nginx default page.
```

Comment:
Do you have to wait 20 minutes deploying with the docker compose, or just with Azure?